presto-tests/src/main/java/io/prestosql/tests/AbstractTestQueries.java

```diff
@@ -842,6 +842,22 @@ public void testLimitAll()
         assertQuery("SELECT custkey, totalprice FROM orders LIMIT ALL", "SELECT custkey, totalprice FROM orders");
     }
 
+    @Test
+    public void testOffset()
+    {
+        String values = "(VALUES ('A', 3), ('D', 2), ('C', 1), ('B', 4)) AS t(x, y)";
+
+        assertQuery("SELECT x FROM " + values + " OFFSET 2 ROWS", "VALUES 'C', 'B'");
```

Comment:
This is not guaranteed to produce `'C', 'B'` since there's no ordering. Verify this query by computing the results and ensuring it contains:
* 2 rows
* they are different from each other
* they are contained in the original set of values

See `testLimit` in this class as an example.